core/src/main/java/com/kin/ecosystem/core/util/DeviceUtils.java

```diff
@@ -43,7 +43,7 @@ private static void checkDensityDpi(DisplayMetrics displayMetrics) {
         int dpi = displayMetrics.densityDpi;
         if (dpi <= DENSITY_HIGH) {
             densityDpi = HDPI;
-        } else if (dpi >= DENSITY_XHIGH && dpi <= DENSITY_XXHIGH) {
+        } else if (dpi >= DENSITY_XHIGH && dpi < DENSITY_XXHIGH) {
```

Comment:
">=" should be ">" as "<=" is used above (doesn't really matter :) )

Reply:
it's not